### PR TITLE
Added graphql args in resolver

### PR DESCRIPTION
**Pull request by hsong-rh**
Related to PR RedHatInsights/insights-api-common-rails#177

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundle
 
 gem 'acts_as_list',        '~> 1.0'
 gem 'faraday',             '>= 0.17.0'
-gem 'insights-api-common', '~> 3.8'
+gem 'insights-api-common', :git => "https://github.com/mkanoor/manageiq-api-common.git", :branch => "support_args"
```
Comment (bzwei) [inline Gemfile:8]: Do we have to use git? can we wait until next gem releases? There is no blocker issue that we have solve by switching to use a git branch.
We can mark the PR wip until the new gem is released.

File: app/controllers/api/v1x0/graphql_controller.rb [hunk 2]
```diff
@@ -16,7 +16,7 @@ def query
       def overlay
         {
           "^.*$" => {
-            "base_query" => lambda do |model_class, _ctx|
+            "base_query" => lambda do |model_class, graphql_args, _ctx|
               policy_scope(model_class.all)
```
Comment (bzwei) [inline app/controllers/api/v1x0/graphql_controller.rb:20]: `graphql_args` is not used.
We should switch to use `policy_scope(model_class)`, shouldn't we?

File: Gemfile (hunk 1 above)
Comment (mkanoor) [reply Gemfile:8]: @bzwei That is going to get stored in the user_context, we just wanted to get level set with the manager-common-gem changes and then start using the graphql_args.

File: app/controllers/api/v1x0/graphql_controller.rb (hunk 2 above)
Comment (hsong-rh) [reply app/controllers/api/v1x0/graphql_controller.rb:20]: yes, the change will be in the next PR.

**Pull request merged**